stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/ChannelListActivity.kt

```diff
@@ -0,0 +1,49 @@
+package io.getstream.chat.android.ui.channel
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import io.getstream.chat.android.ui.R
+import io.getstream.chat.android.ui.databinding.StreamUiFragmentContainerBinding
+
+/**
+ * An Activity representing a self-contained channel list screen. This Activity
+ * is simply a thin wrapper around [ChannelListFragment].
+ */
+public open class ChannelListActivity : AppCompatActivity() {
+    private lateinit var binding: StreamUiFragmentContainerBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = StreamUiFragmentContainerBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        if (savedInstanceState == null) {
+            supportFragmentManager.beginTransaction()
+                .replace(R.id.container, createChannelListFragment())
+                .commit()
+        }
+    }
+
+    /**
+     * Creates an instance of [ChannelListFragment]. Override this method if you want to create an
+     * instance of [ChannelListFragment] with custom arguments or if you want to create a subclass
+     * of [ChannelListFragment].
+     */
+    protected open fun createChannelListFragment(): ChannelListFragment {
+        return ChannelListFragment.newInstance {
+            setFragment(ChannelListFragment())
+            setTheme(R.style.StreamUiTheme_ChannelListScreen)
```

Comment:
This theme is only necessary to: 
- hide "View info" button in the channel options dialog
- hide action button in the channel list header  
Maybe better to make then hidden by default but it is a braking change